infra/modules/gcp-tf-runner/main.tf

```diff
@@ -30,6 +30,11 @@ data "google_service_account_id_token" "identity" {
   target_audience = "worklytics.co/gcp-tf-runner"
 }
 
+# as this runs `terraform console` underneath, it is hacky and SLOW!!!
+data "external" "identity" {
```

Comment:
thinking something like this should exist just found this -> https://registry.terraform.io/providers/adminsentecacommerce/tfvars/latest

Not good examples on how to use though

Reply:
Also this -> https://terraform-docs.io/reference/tfvars-json/

Reply:
That would be fine for internal-use, but we need to minimize dependencies, esp those published by small 3rd parties.